x-pack/plugins/apm/server/lib/services/get_services/get_services_items.ts

```diff
@@ -19,72 +20,315 @@ import {
 } from '../../helpers/setup_request';
 import { getServicesProjection } from '../../../../common/projections/services';
 
+const MAX_NUMBER_OF_SERVICES = 500;
```

Comment:
Shouldn't we have a settings option instead of hardcoded?

Reply:
@sqren thoughts here? We didn't have one before AFAIK.

Reply:
I think there are arguments for making it configurable but no-one ever asked for this and I'd rather not make it configurable if it's not needed so... let's leave it for now, and change it if someone asks for it.